scripts/delete-default-gateway-routes.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+PROJECT_ID=$1
+NETWORK_ID=$2
+FILTERED_ROUTES=$(gcloud compute routes list \
```

Comment:
The `test/ci_integration.sh` script should be the source of truth for how those environment variables are set (because that's the script that gets called out of the `make test_integration` target).